core-server/src/main/java/org/orbisgis/orbisserver/GetCapabilitiesController.java

```diff
@@ -0,0 +1,111 @@
+ *
+ * This file is part of OrbisGIS.
+ *
+ * OrbisServer is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * OrbisServer is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with
+ * OrbisServer. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, please consult: <http://www.orbisgis.org/>
+ * or contact directly:
+ * info_at_ orbisgis.org
+ */
+
+package org.orbisgis.orbisserver;
+
+ import org.orbiswps.server.WpsServer;
+ import org.orbiswps.server.model.JaxbContainer;
+ import org.wisdom.api.DefaultController;
+ import org.wisdom.api.annotations.Controller;
+ import org.wisdom.api.annotations.Parameter;
+ import org.wisdom.api.annotations.Route;
+ import org.wisdom.api.annotations.View;
+ import org.wisdom.api.http.HttpMethod;
+ import org.wisdom.api.http.Result;
+ import org.wisdom.api.templates.Template;
+
+ import javax.xml.bind.JAXBElement;
+ import javax.xml.bind.JAXBException;
+ import javax.xml.bind.Marshaller;
+ import javax.xml.bind.Unmarshaller;
+
+ /**
+  * Instance of DefaultController containing a WpsServer.
+  *
+  * @author Guillaume MANDE
+  */
+ @Controller
+ public class GetCapabilitiesController extends DefaultController {
+
+   /**
+    * IndexController's object, used to get some useful attributes.
+    */
+   private IndexController indexController = new IndexController();
+
+   /**
+    * The action method returning the xml file corresponding to the GetCapabilities method. It handles
+    * HTTP GET request on the "/orbisserv/ows" URL.
+    *
+    * @Parameter the service
+    * @Parameter the version
+    * @Parameter the request GetCpabilities
+    *
+    * @return the xml file
+    */
+   @Route(method = HttpMethod.GET, uri = "/orbisserver/ows")
+   public Result test(@Parameter("service") String service, @Parameter("version") String version, @Parameter("request") String request) {
+       //Simple example of getting information from the WpsServer
+       if(service != null && !service.isEmpty()){
+         if(service.equals("WPS")){
+           if(version != null && !version.isEmpty()){
+             if(version.equals("2.0.0")){
+               if(request!= null && !request.isEmpty()){
+                 if(request.equals("GetCapabilities")){
+                   try { indexController.simpleWpsRequest();  } catch (JAXBException ignored) {}
+                   return ok(indexController.wpsCapabilitiesType);
+                 }else{
+                   return badRequest("The request was not properly written");
```

Comment:
I have a doubt about how to use exceptions listed in the wps standard, should i do like in orbiswps with an ExceptionType ?